strawberryfields/gbs/plot.py

```diff
@@ -360,14 +360,13 @@ def spectrum(
     Returns:
          Figure: spectrum in the form of a histogram of energies with a Lorentzian-function curve
     """
+    if len(energies) < 2:
+        raise ValueError("Number of sampled energies must be at least two")
```

Comment:
I would prefer to keep the try-except import as the very first thing, just like how in a script the imports are the first thing

Reply:
I had that initially but got an error (I believe it was CodeFactor) complaining about the order. So brought this condition to the beginning.